e2e/gui/src/test/java/com/epam/pipeline/autotests/ao/Configuration.java

```diff
@@ -327,4 +328,8 @@ public static Runnable confirmConfigurationChange() {
         return () -> new ConfirmationPopupAO<>(new Object()).ensureTitleIs("Are you sure you want to change configuration?").ok();
     }
 
+    public Configuration assertPageTitleIs(final String expectedTitle) {
+        assertTrue($(title()).toString().contains(expectedTitle));
```

Comment:
It would be better get $(title()).getText()

Reply:
Licence header of file should be updated according to changes

Reply:
Done